### ISSUE TITLE
Header row issue

### ISSUE THREAD

**Issue by VietDN** [6pac/SlickGrid#234]
I would like to confirm this issue is a bug or not while I am upgrading to 2.3.16.
I found these lines of code were wrong 
`var idx = (typeof columnIdOrIdx === "number" ? columnIdOrIdx : getColumnIndex[columnIdOrIdx]);`
on the following functions
```
getHeaderColumn
getHeaderRowColumn
getFooterRowColumn

```
The getColumnIndex was a function and defined as
```
function getColumnIndex(id) {
      return columnsById[id];
}
```

And it was call on some where like this
```
function updateColumnHeader(columnId, title, toolTip) {
      if (!initialized) { return; }
      var idx = getColumnIndex(columnId);
...
```

Please correct me if I missed anything.
Thanks.

**Comment by ghiscoding** [comment]
Probably related to #216 which was fixed in latest version `2.3.17`

**Comment by VietDN** [comment]
Thank ghiscoding
